subprojects/gradle/module-dependencies-graph/src/gradleTest/kotlin/com/avito/module/dependencies/DependenciesGraphTestProject.kt

```diff
@@ -57,15 +57,15 @@ internal object DependenciesGraphTestProject {
                         ),
                     )
                 ),
-                AndroidLibModule(name = "LeafC", dependencies = setOf(GradleDependency.Safe.project(":LeafD"))),
-                AndroidLibModule(name = "LeafG", dependencies = setOf(GradleDependency.Safe.project(":LeafE"))),
-                KotlinModule(name = "LeafD", dependencies = setOf(GradleDependency.Safe.project(":LeafE"))),
```

Comment:
In graph theory "leaf" node means no dependencies. 
This distinction also helps to understand the structure easier in tests.